images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -27,16 +27,25 @@ done
 
 # Get list of all released SDKs from channels which are not end-of-life or preview
 sdks=()
-for version in ${DOTNET_VERSIONS[@]}; do
-    release_url="https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/${version}/releases.json"
-    download_with_retries "${release_url}" "." "${version}.json"
-    releases=$(cat "./${version}.json")
-    sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdk.version'))
-    sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version'))
-    rm ./${version}.json
+for dotnet_version in ${DOTNET_VERSIONS[@]}; do
+    release_url="https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/${dotnet_version}/releases.json"
+    download_with_retries "${release_url}" "." "${dotnet_version}.json"
+    releases=$(cat "./${dotnet_version}.json")
+    versions=()
+    versions=("${versions[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdk.version'))
+    versions=("${versions[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version'))
+    versions_sorted=$(echo ${versions[@]} | tr ' ' '\n' | grep -v preview | grep -v rc | grep -v display | cut -d\" -f2 | sort -u)
+
+    unset features
+    declare -A features
+    for version in $versions_sorted; do
+        feature=$(echo $version|grep -oP '\d+\.\d+\.\d')
+        features[$feature]=$version
+    done
+    sdks=("${sdks[@]}" $(echo ${features[@]}))
+    rm ./${dotnet_version}.json
 done
 
-sortedSdks=$(echo ${sdks[@]} | tr ' ' '\n' | grep -v preview | grep -v rc | grep -v display | cut -d\" -f2 | sort -u -r)
```

Comment:
I think the second loop can be replaced with the `| sort -r | uniq -w 5` commands, like this:
```
sortedSdks=$(echo ${sdks[@]} | tr ' ' '\n' | grep -v preview | grep -v rc | grep -v display | cut -d\" -f2 | sort -r | uniq -w 5)
```